packages/terra-dialog-modal/src/terra-dev-site/test/dialog-modal/DiaglogModalWithSelect.test.jsx

```diff
@@ -49,7 +49,7 @@ class DefaultDialogModal extends React.Component {
           footer={<ActionFooter start="Footer Goes here" />}
         >
           <p>{text}</p>
-          <Select placeholder="Select a color" className={cx('selection-list')}>
+          <Select variant="combobox" placeholder="Select a color" className={cx('selection-list')}>
```

Comment:
Can we rename this file DialogModalWithSelect?  I just noticed the typo.

Reply:
Done